ansible/roles/postgresql/defaults/main.yml

```diff
@@ -1,6 +1,7 @@
 pg_install: True
 postgresql_version: '9.1'
 postgresql_port: 5432
+postgresql_ssl_enabled: True
```

Comment:
I think we want to default this to False now as we don't us postgres SSL, correct? I thought we already made this change when adding pgbouncer, as that doesn't use SSL to connect.